docs/website/views/_layouts/popular-articles.njk

```diff
@@ -2,8 +2,7 @@
 layout: content
 ---
 
-<h2 class="popular-articles-title">Welcome on Exadel authoring Kit for AEM</h2>
-<h3 class="popular-articles-subheader">Popular articles</h3>
+<h2 class="popular-articles-subheader">Welcome on Exadel authoring Kit for AEM</h2>
```

Comment:
#### Considering previous change, this page wasn't looking well. So I thought I should include this update here. Let me know what you think about this changes

_Before update_
![1](https://user-images.githubusercontent.com/86774452/181739329-db325499-e7ee-444c-bf73-dda7cf474cbd.PNG)

_After_ 
![2](https://user-images.githubusercontent.com/86774452/181739341-a44a89d8-76f3-4d2e-8d33-4f33aaf3dfcf.PNG)

Reply:
maybe we can remove this line `Welcome on Exadel authoring Kit for AEM` altogether?

Reply:
I agree with your suggestion)